static/app.js

```diff
@@ -154,27 +193,34 @@ $(function() {
             }  
         }
         else {
-            $('.results').empty()
-            term = $('#term').val()
-            try {
-                const response = await axios({
-                    url: `${base_url}search/`,
-                    method: "GET",
-                    params: {
-                        query: term
-                    }
-                })
-                Object.entries(response.data).forEach((entry) => {
-                    const [key, value] = entry;
-                    const brewery = value
-                    renderBrewery(brewery)
-                })
+             searchByKeyword()
             }   
-            catch(e) {
-                console.log(e.message)
-            }
-        }    
-    })
+    }   
+
+    async function searchByKeyword() {
```

Comment:
JSDoc comment here could be useful for future contributors

Reply:
done,  added comments to all of app.js